src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -1878,6 +1871,53 @@ have to be updated for changes to the rules above, or to support more deeply nes
 	display: none;
 }
 
+.interactive-item-container.interactive-request {
+	align-items: flex-end;
+	padding-bottom: 0px;
+}
+
+.interactive-item-container.interactive-request .value .rendered-markdown  {
+	background-color: var(--vscode-editor-foldBackground);
+	border-radius: 8px;
+	padding: 8px 12px;
+	max-width: 90%;
+	margin-left: auto;
+	width: fit-content;
+	margin-bottom: 5px;
+	box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+	position: relative;
+}
+
+.interactive-item-container.interactive-request .value .rendered-markdown > :last-child {
+	margin-bottom: 0;
+}
+
+.interactive-item-container.interactive-request .value .rendered-markdown {
+	margin-left: auto;
+}
+
+.interactive-item-container.interactive-request .value > .rendered-markdown p {
+	margin-left: auto;
+	width: fit-content;
+}
+
+.interactive-session .interactive-item-container.interactive-request .chat-attached-context {
```

Comment:
I see this rule here https://github.com/microsoft/vscode/blob/f932a599ddaf629cb5d516fc75859e1576739b97/src/vs/workbench/contrib/chat/browser/media/chat.css#L1313 you can delete the other obsolete margin-top